t/lib/OpenQA/Test/Utils.pm

```diff
@@ -250,7 +248,7 @@ sub create_websocket_server {
 }
 
 sub create_scheduler {
-    my ($port) = @_;
+    my ($port, $no_stale_job_detection) = @_;
```

Comment:
This doesn't seem to be used anywhere?

Reply:
Right. I just moved this flag and it might be generally useful but it turns out it is currently not needed. I can remove it if you like.

Reply:
it's used in L262

Reply:
Yes, but the function is never called with that parameter.